src/Security/OpenIdLoginAuthenticator.php

```diff
@@ -47,7 +47,7 @@ public function supports(Request $request)
     /**
      * @throws ValidationException
      */
-    public function getCredentials(Request $request)
+    protected function getCredentials(Request $request)
```

Comment:
I would prefer a rename to `getClaims` as you suggested on Teams

Reply:
Fixed in https://github.com/itk-dev/openid-connect-bundle/pull/12/commits/c4ad38b489660ded6cca2d13e8f2ca030c1a83be